pkg/ebpf/tracer.go

```diff
@@ -81,7 +82,12 @@ func NewFlowFetcher(
 		}
 		return nil, fmt.Errorf("loading and assigning BPF objects: %w", err)
 	}
-
+	/*
+	 * since we load the program only when the we start we need to release
+	 * memory used by cached kernel BTF see https://github.com/cilium/ebpf/issues/1063
+	 * for more details.
+	 */
+	btf.FlushKernelSpec()
```

Comment:
@msherif1234 do we also need to explicitly call the GC here after flushing kernel data structures as per your cilium cilium issue?

Reply:
I am handling the flushing as well as adding gomemlimit in https://github.com/netobserv/netobserv-ebpf-agent/pull/140 maybe I should merge them in one PR